main/zoe/api/zoe-helpers.md

```diff
@@ -236,3 +283,62 @@ const sell = seat => {
   return 'Trade Successful';
 };
 ```
+ 
+## depositToSeat(zcf, recipientSeat, amounts, payments)
+- `zcf` `{ContractFacet }`
+- `recipientSeat` `{ZCFSeat}`
+- `amounts` `{AmountKeywordRecord}`
+- `payments` `{PaymentPKeywordRecord}`
+- Returns: `{Promise<string>}`
+Deposit payments such that their amounts are reallocated to a seat.
+The `amounts` and `payments` records must have corresponding
+keywords.
+
+If the seat has exited, aborts with the message `The recipientSeat cannot have exited.`
+
+On success, returns the exported and settable `depositToSeatSuccessMsg` which
+defaults to `Deposit and reallocation successful.`
+```js
+import {
+  depositToSeat,
+} from '@agoric/zoe/src/contractSupport';
+await depositToSeat(zcf, zcfSeat, { Dep: quatloos(2) }, { Dep: newQuatloos });
+```
+
+## withdrawFromSeat(zcf, seat, amounts)
+- `zcf` `{ContractFacet }`
+- `seat` `{ZCFSeat}`
+- `amounts` `{AmountKeywordRecord}`
+- Returns: `{Promise<PaymentPKeywordRecord>}`
+
+Withdraw payments from a seat. Note that withdrawing the amounts of
+the payments must not and cannot violate offer safety for the seat. The
+`amounts` and `payments` records must have corresponding keywords.
+
+If the seat has exited, aborts with the message `The recipientSeat cannot have exited.`
+
+Unlike `depositToSeat()`, there is no message on a successful completion.
+```js
+import {
+  withdrawFromSeat,
+} from '@agoric/zoe/src/contractSupport';
+const promises = await withdrawFromSeat(zcf, zcfSeat, { With: quatloos(2) });
+```
+
```

Comment:
I think this is missing the point, which is that it returns a promise for a paymentPKeywordRecord

Reply:
So maybe change "promises" to "paymentKeywordRecord"? Otherwise not sure what could be done to make that point other than adding a comment?

Reply:
I think that makes sense to change, also instead of `there is no message on a successful completion` let's say `a paymentKeywordRecord is returned, not a success message`